ios/MockCode/Info.plist

```diff
@@ -38,7 +38,7 @@
 	<key>NSBluetoothPeripheralUsageDescription</key>
 	<string>Used for pairing with nearby devices</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string></string>
+	<string/>
```

Comment:
This looks entirely useless. What is ios doing. It used to just be an empty string field, now it's definitely not anything useful since it's the single tag style. Are we just going to ignore it? Assume it has an idea of what it's doing?